vllm/distributed/kv_transfer/kv_pipe/base.py

```diff
@@ -7,11 +7,11 @@
 class KVPipeBase(ABC):
 
     @abstractmethod
-    def send_tensor(self, tensor: Optional[torch.Tensor]) -> None:
+    def send_tensor(self, tensor: Optional[torch.Tensor], tensor_key: str = "") -> None:
```

Comment:
Adding `tensor_key` is definitely needed for DBs. Would be great if you can make it Optional[str] to force people to generate this metadata if their implementation correctness is based on correct `tensor_key` .